src/core/server/http/http_config.ts

```diff
@@ -31,6 +32,15 @@ const configSchema = schema.object(
         validate: match(validBasePathRegex, "must start with a slash, don't end with one"),
       })
     ),
+    shutdownTimeout: schema.duration({
+      defaultValue: '30s',
+      validate: (duration) => {
+        const durationMs = duration.asMilliseconds();
+        if (durationMs < 1000 || durationMs > 2 * 60 * 1000) {
```

Comment:
I'm surprised `@kbn/config-schema` doesn't provide `min` and `max` in `DurationOptions` https://github.com/elastic/kibana/blob/4584a8b570402aa07832cf3e5b520e5d2cfa7166/packages/kbn-config-schema/src/types/duration_type.ts#L18-L22
Would you mind filing an issue?

Reply:
Done! https://github.com/elastic/kibana/issues/97513